blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
+        function() {
+            var code = myself.stringify();
+            window.prompt('scratchblocks code for you to copy and paste', code);
```

Comment:
okay, how about replacing this with a Morphic dialog box? Now that Morphic supports copy & paste of text, we can do this nicer :-)